curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5633.md

```diff
@@ -7,9 +7,9 @@ dashedName: step-9
 
 # --description--
 
-Create a `getDifficultyLevel` function that takes `cookingTime` as a parameter.
+Create a `getDifficultyLevel` function that takes number indicating cooking time as a parameter.
 
-If `cookingTime` is less than `30`, the function should return `"easy"`. If it is less than or equal to `60`, the function should return `"medium"`. Otherwise, the function should return `"hard"`.
+If cooking time is less than or equal to `30`, the function should return `"easy"`. If it is less than or equal to `60`, the function should return `"medium"`. Otherwise, the function should return `"hard"`.
```

Comment:
```suggestion
If the cooking time is less than or equal to `30`, the function should return `"easy"`. If it is less than or equal to `60`, the function should return `"medium"`. Otherwise, the function should return `"hard"`.
```